src/Test/TestCases.Activities/AsyncTaskCodeActivityTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Activities;
+using System.Threading.Tasks;
+using TestObjects.CustomActivities;
+using Xunit;
+
+namespace TestCases.Activities
+{
+    public sealed class AsyncTaskCodeActivityTests
+    {
+        private static readonly AsyncTaskActivity<object> genericActivity = new(Task.FromResult<object>(null));
```

Comment:
This is only used in one test, so it should be declared there.
Also, I don't see any test for the non generic version.